crates/re_space_view_time_series/src/space_view_class.rs

```diff
@@ -277,7 +277,14 @@ impl SpaceViewClass for TimeSeriesSpaceView {
 
         let scalar_axis =
             ViewProperty::from_archetype::<ScalarAxis>(blueprint_db, ctx.blueprint_query, view_id);
-        let y_range = scalar_axis.component_or_fallback::<Range1D>(ctx, self, state)?;
+        let mut y_range = scalar_axis.component_or_fallback::<Range1D>(ctx, self, state)?;
+        if y_range.start() >= y_range.end() {
+            // Ensure that the range is valid - egui_plot might debug_assert otherwise.
+            // `next_up_f64` should be sufficient, but empirically it's not always enough
+            // (likely we're loosing precision due to some calculations down the line)
+            *y_range.end_mut() = y_range.start() + 1.0;
+        }
```

Comment:
Why does `component_or_fallback::<Range1D>` return an inverted range in the first place?

Reply:
The range I see in the debugger is `(3.0000000000000004, 3.0)`. Where does these values come from?

Reply:
Found it: 
https://github.com/rerun-io/rerun/blob/9c4df0b8abd62929915ded783e377285a716ac0c/crates/re_space_view_time_series/src/space_view_class.rs#L660-L660

Reply:
I'll clean this up